metadata_layer.py

```diff
@@ -621,10 +621,11 @@ def mirror_link(self, model_object, data_source, link, link_obj, policy):
 
         # Determine the best URL to use when mirroring this
         # representation.
-        if title and link.rel == Hyperlink.OPEN_ACCESS_DOWNLOAD:
+        if link.rel == Hyperlink.OPEN_ACCESS_DOWNLOAD:
+            url_title = title or identifier.identifier
```

Comment:
During development this was causing the link to not be mirrored if no title was provided. The book really needs to have a title to become a Work, but we can still come up with a URL for it even without a title.